scripts/run.sh

```diff
@@ -1,13 +1,28 @@
 #!/bin/bash
 
+# This must precede setting LD_LIBRARY_PATH below.
+if [[ ! -z "$JENKINS_OPT_ADDITIONAL_PLUGINS" ]]; then
+    echo "Installing additional plugins..."
+    export REF=/var/jenkins_home
+    export JENKINS_WAR=${JENKINS_FOLDER}/jenkins.war
+    CURL_OPTIONS="-fkL" /usr/local/bin/install-plugins.sh $JENKINS_OPT_ADDITIONAL_PLUGINS 2>&1
+    echo "Completed installing additional plugins..."
+else
+    echo "No additional plugins requested for installation..."
+fi
+
 export LD_LIBRARY_PATH=/libmesos-bundle/lib:/libmesos-bundle/lib/mesos:$LD_LIBRARY_PATH
```

Comment:
This should not be required anymore. The new plugin based on USI does not use `libmesos`.